scipy/special/tests/test_mpmath.py

```diff
@@ -1211,17 +1221,15 @@ def test_hermite(self):
 
     # hurwitz: same as zeta
 
-    @nonfunctional_tooslow
     def test_hyp0f1(self):
         assert_mpmath_equal(sc.hyp0f1,
                             _exception_to_nan(lambda a, x: mpmath.hyp0f1(a, x, **HYPERKW)),
-                            [Arg(), Arg()])
+                            [Arg(-10, 1e7), Arg(0, 1e5)])
 
-    @nonfunctional_tooslow
     def test_hyp0f1_complex(self):
         assert_mpmath_equal(lambda a, z: sc.hyp0f1(a.real, z),
                             _exception_to_nan(lambda a, x: mpmath.hyp0f1(a, x, **HYPERKW)),
-                            [Arg(), ComplexArg()])
+                            [Arg(1e-5, 1e7), ComplexArg(0, 1e5)])
```

Comment:
The complex-argument bessel comes from amos. I'm not aware of other complex-valued floating point implementations.

Are you sure the inaccuracy is from the scipy side; mpmath sometimes produces wrong results if dps is too small?

It may also be that the values computed by amos are accurate, and the inaccuracy is due to loss of precision in complex multiplication. If so, this might potentially be avoided by suitable transformations (or series expansion in the problematic regime if it's localized).